ts/screens/onboarding/NewOnboardingEmailInsertScreen.tsx

```diff
@@ -0,0 +1,287 @@
+/**
+ * A screen where user after login (with CIE) can set email address if it is
+ * not present in the profile.
+ */
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { EmailString } from "@pagopa/ts-commons/lib/strings";
+import * as E from "fp-ts/lib/Either";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import { Content } from "native-base";
+import * as React from "react";
+import { useCallback, useMemo, useState } from "react";
+import { View, Keyboard, SafeAreaView, StyleSheet, Alert } from "react-native";
+import validator from "validator";
+import { IOColors, Icon, LabelSmall } from "@pagopa/io-app-design-system";
+import { StackActions } from "@react-navigation/native";
+import { Alert as AlertComponent } from "../../components/Alert";
+import { H1 } from "../../components/core/typography/H1";
+import { VSpacer } from "../../components/core/spacer/Spacer";
+import { LabelledItem } from "../../components/LabelledItem";
+import LoadingSpinnerOverlay from "../../components/LoadingSpinnerOverlay";
+import BaseScreenComponent, {
+  ContextualHelpPropsMarkdown
+} from "../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import I18n from "../../i18n";
+import { IOStackNavigationRouteProps } from "../../navigation/params/AppParamsList";
+import { OnboardingParamsList } from "../../navigation/params/OnboardingParamsList";
+import { profileLoadRequest, profileUpsert } from "../../store/actions/profile";
+import { useIODispatch, useIOSelector } from "../../store/hooks";
+import {
+  profileEmailSelector,
+  profileSelector
+} from "../../store/reducers/profile";
+import { withKeyboard } from "../../utils/keyboard";
+import { areStringsEqual } from "../../utils/options";
+import { Body } from "../../components/core/typography/Body";
+import { IOStyles } from "../../components/core/variables/IOStyles";
+import ROUTES from "../../navigation/routes";
+import { emailInsert } from "../../store/actions/onboarding";
+
+type Props = IOStackNavigationRouteProps<
+  OnboardingParamsList,
+  "NEW_ONBOARDING_INSERT_EMAIL_SCREEN"
+>;
+
+const styles = StyleSheet.create({
+  flex: {
+    flex: 1
+  }
+});
+
+const EMPTY_EMAIL = "";
+
+const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
+  title: "email.insert.help.title",
+  body: "email.insert.help.content"
+};
+
+/**
+ * A screen to allow user to insert an email address.
+ */
+const NewOnboardingEmailInsertScreen = (props: Props) => {
+  const dispatch = useIODispatch();
+
+  // TODO: change this state logic and name (this value will be retrive by the backend)
```

Comment:
Could we provide the JIRA ticket in this comment, changing it to `// FIXME - <jira ticket reference>`?

Reply:
done